ksqldb-common/src/main/java/io/confluent/ksql/function/UdfFactory.java

```diff
@@ -79,6 +80,10 @@ public String toString() {
         + '}';
   }
 
+  public synchronized KsqlScalarFunction getUdfFunction(final List<SqlArgument> argTypes) {
+    return udfIndex.getUdfFunction(argTypes);
+  }
+
   public synchronized KsqlScalarFunction getFunction(final List<SqlType> argTypes) {
```

Comment:
Currently still used for testing purposes, _should_ be removable after tests are updated